XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotTypePageSpecific.cs

```diff
@@ -166,6 +166,15 @@ public policy_backup_type BackupType
 
         private void RefreshTab(IVMPolicy policy)
         {
+            /* when a policy does not have any VMs, irrespective of
+             * the snapshot type, enable Quiesce 
+             */
+
+            if (policy.VMs.Count == 0)
```

Comment:
You can write this in a more compact way:
quiesceCheckBox.Enabled = (policy.VMs.Count == 0);